docs/clients/logstash/README.md

```diff
@@ -0,0 +1,98 @@
+# Logstash
+
+Loki has a [Logstash](https://www.elastic.co/logstash) output plugin called
+`logstash-output-grafana-loki` that enables shipping logs to a private Loki
+instance or [Grafana Cloud](https://grafana.com/oss/loki).
+
+## Installation
+
+```
+$ bin/logstash-plugin install logstash-output-grafana-loki
+```
+
+## Usage
+In your Logstash configuration, use `output loki`. The configuration values would look like this.
+
+```
+output {
+  loki {
+    [url => "" | default = none | required=true]
+
+    [include_labels => array | default = [] | required=true]
+    
+    [extra_labels => hash | default = {} |  required=false]
+
+    [tenant_id => string | default = nil | required=false]
+
+    [message_field => string | default = "message" | required=false]
+
+    [batch_wait => number | default = 1(s) | required=false]
+
+    [batch_size => number | default = 102400(bytes) | required=false]
+
+    [min_delay => number | default = 1(s) | required=false]
+
+    [max_delay => number | default = 300(s) | required=false]
+
+    [retries => number | default = 10 | required=false]
+
+    [username => string | default = nil | required=false]
+    
+    [password => secret | default = nil | required=false]
+
+    [cert => path | default = nil | required=false]
+
+    [key => path | default = nil| required=false]
+
+    [ca_cert => path | default = nil | required=false]
+
+  }
+}
+```
+
+## Docker Image
+
+There is a Docker image `grafana/logstash-output-grafana-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/logstash/conf)
+
+## Configuration
+
+### url
+The url of the Loki server to send logs to.  When sending data the publish path (`/api/loki/v1/push`) will automatically be appended.
```

Comment:
Let's not append the path. 

1 - This is more flexible
2 - This is consistent with other client.